### ISSUE TITLE
The user must only be able to select dates that fall during the Event while creation of jobs

### ISSUE THREAD

**Issue by anjali-dhanuka** [systers/vms#655]
## Description
As a user I must be able to select only those dates for a job that fall during the corrresponding event so that the user doesn't encounter the error at last.

## Mocks
![screenshot from 2018-03-06 11-35-01](https://user-images.githubusercontent.com/22369767/37016498-80c77fbe-2132-11e8-98af-22cb520477e0.png)

### Update [Required]
- [ ] Set minimum date and maximum date in datepicker

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
10 minutes

**Comment by anjali-dhanuka** [comment]
I want to work on the issue @abhishekarya286 @ramitsahwney @prachi1210 @tapasweni-pathak @tonythomas01.

**Comment by abhishekarya286** [comment]
Hi Anjali,
Please go ahead.

**Comment by LordGameleo** [comment]
Is this issue still open? I would like to work on it. @abhishekarya286  @prachi1210  @tonythomas01

**Comment by Monal5031** [comment]
GFI @LordGameleo :+1:

**Comment by kss682** [comment]
Hi , I am new here I would like to work on this issue if its still open .

**Comment by Monal5031** [comment]
ping @kss682

**Comment by abha224** [comment]
@kss682 Are you still working on this issue? Pls let us know so that we can open it up for others, in case you aren't.

**Comment by kss682** [comment]
@abha224 Sorry I couldn't respond earlier ,unassigning myself .

**Comment by abha224** [comment]
@kss682 okay :+1:

**Comment by decon-harsh** [comment]
@sanchibansal340 Can I work on this , I actually made the necessary changes I guess it's working on my local .

**Comment by sanchibansal340** [comment]
@decon-harsh Were you assigned the issue?

**Comment by decon-harsh** [comment]
Nope Can you assign me now?

**Comment by sanchibansal340** [comment]
You will have to wait till Open Source Hack starts on 1st September.

**Comment by decon-harsh** [comment]
Can I take this up now?

**Comment by sanchibansal340** [comment]
@decon-harsh You have been assigned another issue and one person can be assigned only 1 issue. You can submit PR for the issue assigned to you then I will assign this to you :)

**Comment by decon-harsh** [comment]
Hey @sanchibansal340 Can I get this now?

**Comment by Anmollenka** [comment]
Can I work on this issue as a part of OSH??Thank You.

**Comment by sanchibansal340** [comment]
@decon-harsh You have been assigned another issue. Since you said that this is working locally for you, would you like me to assign this to you and maybe you can unassign yourself from the other issue for the time being and create a PR real quick?

**Comment by decon-harsh** [comment]
Created a PR on the last assigned issue and it is in review. So, I can work on this!

**Comment by yxlin118** [comment]
Can I work on this issue for OSH? Thanks!

**Comment by codesankalp** [comment]
I want to work on this issue. @anjali-dhanuka

**Comment by sanchibansal340** [comment]
@Anmollenka and @decon-harsh You have been assigned another issue.
@yxlin118 I am assigning it to you.

**Comment by yxlin118** [comment]
![image](https://user-images.githubusercontent.com/54916304/92043224-8394c680-ed30-11ea-99fb-e685d2424b28.png)
@sanchibansal340 I am getting this error when trying to set up, do you happen to know how to fix this?

**Comment by yxlin118** [comment]
![image](https://user-images.githubusercontent.com/54916304/92043951-00747000-ed32-11ea-8320-9b774aea1238.png)
I got this error even after I followed the readme to set the secret key

**Comment by sanchibansal340** [comment]
@yxlin118 You can ask setup related doubts on Zulip chat

**Comment by sanchibansal340** [comment]
@yxlin118 Were you able to setup the project successfully?

**Comment by yxlin118** [comment]
@sanchibansal340 No I am still trying to figure out. How do I join Zulip? Thank you for your help!

**Comment by sanchibansal340** [comment]
You can join [Zulip](https://anitab-org.zulipchat.com/login/) from here. Hope this helps 💯

**Comment by decon-harsh** [comment]
ping @yxlin118 Are you still working on this?

**Comment by yxlin118** [comment]
@decon-harsh Yes but I am currently still stuck on the set up:(

**Comment by sanchibansal340** [comment]
@yxlin118 Stuck with the setup?

**Comment by yxlin118** [comment]
@sanchibansal340 yes, I tried asking on Zulip and googling and still couldn't figure out:( maybe it's best to reassign this to someone else and I can take another issue after I resolve it

**Comment by sanchibansal340** [comment]
@yxlin118 Did you try asking it in the vms stream?

**Comment by yxlin118** [comment]
@sanchibansal340 Oh yes I forgot, I just did! If I am still not able to figure it out after someone responded, I 'll let you know. Thanks!

**Comment by decon-harsh** [comment]
@sanchibansal340 If this issue is availiable I would love to work on this!

**Comment by sanchibansal340** [comment]
@yxlin118 Updates?

**Comment by yxlin118** [comment]
@sanchibansal340 Sooo sorry, I am still working on the issue with setting up. Maybe it's best to reassign this issue. Thank you for your help and patience!

**Comment by sanchibansal340** [comment]
@decon-harsh I am assigning it to you

**Comment by decon-harsh** [comment]
Need to edit some test cases
